log/logger.go

```diff
@@ -105,8 +117,14 @@ func NewLogger(dst io.Writer, options ...Option) Logger {
 		}
 	}
 
+	var disableFilter *bool
 	if logCfg.Filter != nil {
-		output = NewFilterWriter(output, logCfg.Filter)
+		disableFilter = new(bool)
```

Comment:
what is the effect of calling `new(bool)` here? can't we just replace L120 with this?

Reply:
Sorry, can you look at the latest code. I just changed this